### PR TITLE
fix: add alt text option for images

### PR DESCRIPTION
**Pull request by ellierossuck**
[![PR App][icn]][demo] | Fix RM-4792
:-------------------:|:----------:

## 🧰 Changes

PR 1 of 2: the other part to this fix is in [this PR](https://github.com/readmeio/readme/pull/7182) in the main readme repo.

Previously, we've been setting images' alt tags using the image caption, but we want to provide the option for users to set distinct alt text for their images.

This PR ensures that if we have `alt` set, we'll use that for the image's caption. If not, we'll continue to use the caption.

## 🧬 QA & Testing

I think it will be a bit tricky to test this using PR apps, but if you checkout this branch and also the corresponding branch in the readme repo (`editor/fix/add-alt-text-option-for-images`), link the two repos and run locally, you'll be able to test. 

Here's a screenshot of it working, too. I've inspected the rendered image and you can see the alt text used instead of the caption.
Screen Shot 2022-07-14 at 10 55 11 AM

- [Broken on production][prod].
- [Working in this PR app][demo].

[demo]: https://markdown-pr-PR_NUMBER.herokuapp.com
[prod]: https://SUBDOMAIN.readme.io
[icn]: https://user-images.githubusercontent.com/886627/160426047-1bee9488-305a-4145-bb2b-09d8b757d38a.svg

### CODE REVIEW

File: __tests__/flavored-compilers.test.js [hunk 1]
```diff
@@ -143,6 +143,27 @@ describe('ReadMe Magic Blocks', () => {
...
+    const txt = `[block:image]
+    {
+      "images": [
+        {
+          "image": [
+            "https://files.readme.io/6f52e22-man-eating-pizza-and-making-an-ok-gesture.jpg",
+            "rdme-blue.svg",
+            "A man eating pizza and making an OK gesture"
+          ],
+          "caption": "Ok, __pizza__ man.",
```
Comment (rafegoldberg) [inline __tests__/flavored-compilers.test.js:156]: lol

**Review by rafegoldberg — APPROVED** [review]
Seems sensible!

**Review by kellyjosephprice — APPROVED** [review]
I tested in the PR app with:

```
[block:image]
{
  "images": [
    {
      "image": [
        "http://placekitten.com/g/600/600",
        "title",
        "kitty"
      ],
      "caption": "Testing alt"
    }
  ]
}
[/block]

![kitty](http://placekitten.com/g/600/600 "title")
```

Looks great!

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.46.1</b>](/readmeio/markdown/releases/tag/v6.46.1)
<br>

**Pull request merged**